README.md

```diff
@@ -42,33 +51,35 @@ Installing the Intel(R) MLSL using RPM Package Manager (root mode):
 
         where <version>.<update>-<package#> is a string, such as: 2017.0-009
 
-    3. Uninstalling the Intel(R) MLSL using the RPM Package Manager
+    3. Uninstalling Intel(R) MLSL using the RPM Package Manager
 
         $ rpm -e intel-mlsl-devel-64-<version>.<update>-<package#>.x86_64
 
-Installing the Intel(R) MLSL using install.sh (user mode):
+Installing Intel(R) MLSL using the tar file (user mode):
 
-    1. Run install.sh and follow the instructions.
+        $ tar txf l_mlsl-devel-64-<version>.<update>.<package#>.tgz
```

Comment:
we can't combine "t" and "x" in the same command, maybe just "tar xf <file>" ?

Reply:
I guess it is a misprint, it should be "zxf"

Reply:
Right, it should be zxf. I will update the PR